src/cosmos-copilot.WebApp/Services/SemanticKernelService.cs

```diff
@@ -292,9 +324,15 @@ public async Task LoadProductDataAsync()
     /// <returns>Id of the newly created product item.</returns>
     public async Task<string> InsertProductAsync(Product product)
     {
-        #pragma warning disable SKEXP0020
+
+        // PartitionKey partitionKey = new(product.categoryId);
+        // return await _productContainer.CreateItemAsync<Product>(
+        //     item: product,
+        //     partitionKey: partitionKey
+        // );
+        
```

Comment:
nit: do we need this commented code?

Reply:
I removed it as well as a bunch of other commented out code. I think we're ready to go final since we're now into updating the lab guide.